lib/index.js

```diff
@@ -189,11 +189,11 @@ function adapter(pubsub, opts) {
             return debug('ignore different channel');
         }
 
-        const data = msg.data;
+        const dataMsg = data;
```

Comment:
We can just remove `dataMsg` declaration and use the `data` object declared at 184, right?

Reply:
@idoshamun thank you. Yes, that works as well. I will update the PR.